packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -21,21 +23,22 @@ import {
   CLIError,
   getDefaultUserTerminal,
 } from '@react-native-community/cli-tools';
+import {Device} from '../../types';
 
 type FlagsT = {
-  simulator: string,
-  configuration: string,
-  scheme: ?string,
-  projectPath: string,
-  device: ?(string | true),
-  udid: ?string,
-  packager: boolean,
-  verbose: boolean,
-  port: number,
-  terminal: ?string,
+  simulator: string;
+  configuration: string;
+  scheme: string | void;
```

Comment:
Wouldn't this be more accurate here (and below)?
```suggestion
  scheme?: string;
```